Description: Fix build with v8 3.10
 Node calls V8::Dispose on exit, and that makes v8 crash.
Author: Jérémy Lal <kapouer@melix.org>
Forwarded: https://github.com/joyent/node/issues/3184#issuecomment-6297038
Last-Update: 2012-06-12
--- a/src/node.cc
+++ b/src/node.cc
@@ -2645,7 +2645,7 @@
 #ifndef NDEBUG
   // Clean up.
   context.Dispose();
-  V8::Dispose();
+  // V8::Dispose(); // this call crashes v8 3.10
 #endif  // NDEBUG
 
   return 0;
